src/OpenTelemetry.Instrumentation.Cassandra/README.md

```diff
@@ -0,0 +1,79 @@
+# Cassandra Instrumentation for OpenTelemetry
```

Comment:
CHAGNELOG.md file is needed in the same folder. Please make a copy from https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/1c8cc26cdd1605bb547ae410d4f6da4a5b2309b6/src/OpenTelemetry.Exporter.OneCollector/CHANGELOG.md